insuree/schema.py

```diff
@@ -162,7 +164,13 @@ def resolve_insurees(self, info, **kwargs):
         filters = []
         additional_filter = kwargs.get('additional_filters', None)
         chf_id = kwargs.get('chf_id')
+        chf_id_max_length = 12
```

Comment:
@lruzicki  same here, get that number from the config "insuree_number_length"

Reply:
yeah right, it was never set so i did not use it. I will put it here and add a default that is a max of the filed that it can take if the max-length is not set